relay/kusama/src/governance/origins.rs

```diff
@@ -18,6 +18,7 @@
 
 pub use pallet_custom_origins::*;
 
+#[allow(unreachable_patterns)]
```

Comment:
What warning is this suppressing? I removed them locally and just running clippy seems fine.   
Otherwise please open a bug report in the SDK then, the FRAME macros should not produce warnings.

Reply:
Let's try to remove it and look at CI

Reply:
> Otherwise please open a bug report in the SDK, the FRAME macros should not produce warnings.

Wasn't there a fix for something similar in the SDK pr that upgraded the rust version?

Reply:
https://github.com/paritytech/polkadot-sdk/issues/5697 okay maybe it wasn't fixed. I checked the pr again.